00002_arithmetic.js

```diff
@@ -0,0 +1,49 @@
+/*2. Enter 2 numbers and display the result for the following operations (each operation should be a function)
+    add / multiply / divide / modulus / substract / greater than / less than / is equal.
+*/
+
+const readLine = require("readline-sync");
+const number1 = readLine.question("Enter first Number : ");
+const number2 = readLine.question("Enter the second Number: ");
+
+let addition = (number1, number2) => {
```

Comment:
you should not define a function with let

Reply:
Ok. Changed every function declaration into const.